torchaudio/backend/sox_io_backend.py

```diff
@@ -20,22 +21,24 @@ def __init__(self, sample_rate: int, num_frames: int, num_channels: int):
 
 
 @_mod_utils.requires_module('torchaudio._torchaudio')
-def info(filepath: str) -> AudioMetaData:
+def info(filepath: Union[str, Path]) -> AudioMetaData:
```

Comment:
Could you keep the signature as `str` for torchscript compatibility? (same for the other functions too.)

```suggestion
def info(filepath: str) -> AudioMetaData:
```